contracts/src/libraries/MetaTransaction.sol

```diff
@@ -1,9 +1,25 @@
 // SPDX-License-Identifier: GPL-3.0-only
 pragma solidity ^0.8.30;
 
-/// @title Meta-Transaction Library
-/// @notice Library that defines a meta-transaction type and hash.
+/**
+ * @title Meta-Transaction Library
+ * @notice Library that defines a meta-transaction type and hash.
+ */
 library MetaTransaction {
+    // ============================================================
+    // STRUCTS
+    // ============================================================
+
+    /**
+     * @notice Represents a meta-transaction.
+     * @param chainId The chain ID for the transaction.
+     * @param account The account address initiating the transaction.
```

Comment:
nit: I would write "the account address that is executing the transaction".

For me initiating is slightly different to executing, i.e. the initiating account is the one that sends the transaction to the Safe smart account.